generator/src/main/java/com/linkedin/pegasus/generator/DataSchemaParser.java

```diff
@@ -277,4 +298,50 @@ public ParseResult addMessage(String message)
       return this;
     }
   }
+
+  public static class Builder
```

Comment:
Usually the builder pattern create a new builder each time call a static method.

For example
DataSchemaParser.Builder.<>.<>.build().

Do you think we can do that?

Reply:
There is no convention that builder should have static methods. Infact, the accepted pattern is to use the constructor for the Builder which accepts all required paramters.

Added a static "newBuilder" method for convenience.